frontend/src/pages/Landing/MyAlerts.js

```diff
@@ -13,12 +13,11 @@ import '@trussworks/react-uswds/lib/index.css';
 import './index.css';
 import { ALERTS_PER_PAGE } from '../../Constants';
 import { deleteReport } from '../../fetchers/activityReports';
-import Filter from '../../components/Filter';
-import ReportMenu from './ReportMenu';
 import TooltipWithCollection from '../../components/TooltipWithCollection';
 import Tooltip from '../../components/Tooltip';
+import TableHeader from '../../components/TableHeader';
 
-function ReportsRow({ reports, removeAlert, message }) {
+export function ReportsRow({ reports, removeAlert, message }) {
```

Comment:
exported so we can test it. it may get re-factored away soon, but needed the test coverage